packages/operator/pkg/webserver/routes/v1/packaging/pi_routes.go

```diff
@@ -0,0 +1,36 @@
+//
+//    Copyright 2019 EPAM Systems
```

Comment:
```suggestion
//    Copyright 2020 EPAM Systems
```
Maybe, it's just safer to update in all if it does matter